src/OBD9141sim.cpp

```diff
@@ -48,6 +48,12 @@ void OBD9141sim::initialize(){
     this->initialised = true;
 }
 
+void OBD9141sim::initializeKWP(){
+    use_kwp_ = true;
```

Comment:
```suggestion
    this->use_kwp_ = true;
```

I know it's not necessary, but the other code follows this pattern, so lets keep it similar.